Makefile

```diff
@@ -135,13 +135,9 @@ benchmark-test:
 	go test -run=XX -bench=. ./agent/...
 
 
-.PHONY: build-image-for-ecr ecr-execution-role-image-for-upload upload-images replicate-images
+.PHONY: build-image-for-ecr upload-images replicate-images
 
-build-image-for-ecr: netkitten volumes-test squid awscli image-cleanup-test-images fluentd taskmetadata-validator \
-						testnnp container-health-check-image telemetry-test-image storage-stats-test-image ecr-execution-role-image-for-upload
-
-ecr-execution-role-image-for-upload:
-	$(MAKE) -C misc/ecr-execution-role-upload $(MFLAGS)
+build-image-for-ecr: netkitten volumes-test awscli image-cleanup-test-images fluentd taskmetadata-validator
```

Comment:
are these used for integration tests?

Reply:
These images are used by integ tests, yes.